pkg/netlink/netlink.go

```diff
@@ -218,13 +228,13 @@ func (n *netlinkType) EnsureLooseModeIsConfigured(interfaceName string) error {
 	return fmt.Errorf("loose mode not configured on iface %q", interfaceName)
 }
 
-func (n *netlinkType) EnableForwarding(interfaceName string) error {
-	err := setSysctl(ipv4ConfPath(interfaceName)+"/forwarding", []byte("1"))
+func (n *netlinkType) EnableForwarding(interfaceName string, family k8snet.IPFamily) error {
+	err := setSysctl(ipConfPath(interfaceName, family)+"/forwarding", []byte("1"))
 	return errors.Wrapf(err, "unable to update forwarding on interface %q", interfaceName)
 }
 
 func (n *netlinkType) GetReversePathFilter(interfaceName string) ([]byte, error) {
-	path := ipv4ConfPath(interfaceName) + "/rp_filter"
+	path := ipConfPath(interfaceName, k8snet.IPv4) + "/rp_filter"
```

Comment:
Technically we should elide if v4 here as well however this function isn't called externally so we could just make it package scope.